databricks/koalas/datetimes.py

```diff
@@ -105,14 +105,6 @@ def second(self) -> 'ks.Series':
         """
         return _wrap_accessor_spark(self, F.second, LongType()).alias(self.name)
 
-    @property
-    def millisecond(self) -> 'ks.Series':
-        """
-        The milliseconds of the datetime.
-        """
-        return _wrap_accessor_pandas(
```

Comment:
Why did we remove this? Does this not work?

Reply:
I thought these were the reuses of the dt functions from the Pandas Series.
However, in the screenshots I added to PR, the Pandas dt Property does not support milliseconds.
So I thought this was probably added by mistake.
And It doesn't actually work as you asked, (as in the following screenshot.)

![스크린샷 2019-09-01 오후 2 06 51](https://user-images.githubusercontent.com/44108233/64071997-c6593400-ccc1-11e9-88b0-767e32ec70a0.png)

Reply:
@HyukjinKwon, Thanks to Review.
I thought these were the reuses of the Pandas Series dt Property.
However, in the screenshots I added to PR, the Pandas Series dt Property does not support milliseconds.
So I thought this was probably added by mistake.
And It doesn't actually work as you asked, (as in the following screenshot.)

![스크린샷 2019-09-01 오후 2 06 51](https://user-images.githubusercontent.com/44108233/64071997-c6593400-ccc1-11e9-88b0-767e32ec70a0.png)